charts/logzio-telemetry/values.yaml

```diff
@@ -285,129 +285,7 @@ metricsConfig:
           - memory_limiter
         receivers:
           - prometheus
-
```

Comment:
why we removed this part?

Reply:
`values.config` is not applied in any of the templates. we have [tracesConfig](https://github.com/logzio/logzio-helm/blob/master/charts/logzio-telemetry/values.yaml#L149) and [metricsConfig](https://github.com/logzio/logzio-helm/blob/master/charts/logzio-telemetry/values.yaml#L195) that are responsible for the collector configuration, So `values.config` is redundant and can be confusing for customers that want to modify the chart :)